packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -98,7 +98,8 @@ export interface TreeNodeSchemaClass<
 	out TNode = unknown,
 	in TInsertable = never,
 	out ImplicitlyConstructable extends boolean = boolean,
-> extends TreeNodeSchemaCore<Name, Kind, ImplicitlyConstructable> {
+	out Info = unknown,
```

Comment:
Should we add `@typeParam` docs for this? Or do you think the runtime param documentation is sufficient?

Reply:
What do you mean by "runtime param documentation is sufficient". This is type parameter to a type. There is no runtime here.

Reply:
I should have said property. I'm wondering if there is anything we can say about the `Info` type param that isn't captured by the `info` property.

Reply:
I added a little detail to the field's comment, which I think is enough. No one should really be using this type directly except for the implementation of schemaFactory.